src/main/webapp/js/getAnnotation.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Fetch an annotation from the server when an annotated line is clicked
```

Comment:
Shouldn't this be fetch all annotations for a line from the server?

Reply:
The comment here may be a bit misleading since this will fetch all annotations tied to a particular line. I will make this more clear in the comment